homotopy-core/src/rewrite.rs

```diff
@@ -1005,8 +1003,8 @@ impl Cone {
         index: usize,
         source: Vec<Cospan>,
         target: Cospan,
-        regular_slices: Vec<Rewrite>,
-        singular_slices: Vec<Rewrite>,
+        regular_slices: &[Rewrite],
+        singular_slices: &[Rewrite],
```

Comment:
Could you please revert this change? It's not necessary after my change and it's causing a merge conflict.